packages/server/core/src/types/config/server.ts

```diff
@@ -41,6 +41,9 @@ export type SSR =
       preload?: boolean | SSRPreload;
       inlineScript?: boolean;
       disablePrerender?: boolean;
+      unsafeContext?: {
+        headers?: string[];
```

Comment:
Are there any other fields here?

Reply:
Haven not.
unsafeContext is an Object because of the request have many attributes. 
But I think this fields can be `unsafeHeader`... ?
What about your proposal?